utils/githooks/pre-commit.d/50-clang-format.sh

```diff
@@ -13,12 +14,14 @@ if [ -e .git/MERGE_HEAD ]; then
 fi
 
 if ! command -v git-clang-format > /dev/null 2>&1; then
-    echo "git-clang-format not installed. Skipping"
-    exit 0
+    echo "git-clang-format not installed."
+    echo "See ./utils/githooks/README.md#2-install-the-required-tools for instructions."
+    exit 1
 fi
 if ! command -v clang-format > /dev/null 2>&1; then
```

Comment:
`if ! command -v clang-format > /dev/null 2>&1; then`
I think we can remove this check as:
- there is a strict dependency between `git-clang-format` pkg and `clang-tools-extra` (`clang-format`) pkg. Every time 
- the presence of clang-format is checked by `extra.py` script few lines later